includes/apple-push-api/request/class-request.php

```diff
@@ -241,12 +241,24 @@ private function parse_response( $response, $json = true, $type = 'post', $meta
 			// Add the MIME request.
 			$body .= "\n\n" . esc_html__( 'MIME request', 'apple-news' ) . ":\n" . $debug_mime_request . "\n";
 
+			/**
+			 * Add headers (such as From:) to notification messages.
+			 *
+			 * See https://developer.wordpress.org/reference/functions/wp_mail/ for documentation and examples.
+			 *
+			 * @since 4.3.0
```

Comment:
We'll need to update this to whatever the new version of the plugin ends up being (likely 2.0.0).

Reply:
For now, let's do `x.x.x`.

Reply:
Done.